Resources/templates/CommonAdmin/EditTemplate/fieldsets.php.twig

```diff
@@ -29,7 +29,7 @@
                             <div class="row">
                                 {% for column in rowColumns %}
                                     {% if column.credentials is not empty %}
-                                        {{ echo_if_granted(column.credentials, builder.ModelClass) }}
+                                        {{ echo_if("is_one_admingenerator_granted(" ~ echo_twig_arr([column.credentials]) ~ ", " ~ builder.ModelClass ~ ')') }}
```

Comment:
@sescandell the only problem then is that it will not be encapsulated in an `Expression`